probe/probe.go

```diff
@@ -335,18 +334,48 @@ func (p *Probe) mesureOperation(operationName string, operation func() error) er
 	return nil
 }
 
+func (p *Probe) checkDurabilityBucketHasEnoughObject() (error, bool) {
```

Comment:
In go, the error is usually the second field

Reply:
Done